readme.md

```diff
@@ -363,13 +363,15 @@ test(t => {
 });
 ```
 
-You can also use your own local Babel version:
+To use your own local Babel version, specify the version you want in your package.json, and ensure you're using npm@3 (or run `$ npm dedupe`), so the dependency will be deduped.
+
+*We don't yet [support Babel 6](https://github.com/sindresorhus/ava/pull/221), but you can still use it in your own project.*
 
 ```json
 {
 	"devDependencies": {
 		"ava": "^0.6.0",
-		"babel-core": "^5.8.0"
+		"babel-core": "5.8.11"
```

Comment:
@novemberborn That's not really the point of that example. It could use either, but by using an exact version of Babel 5, I'm illustrating my above point of it being deduped.